src/tests/flyer.test.ts

```diff
@@ -294,4 +293,41 @@ describe('deleteFlyer tests', () => {
     const deleteFlyerResponse = await FlyerRepo.deleteFlyer('64811792f910705ca1a981f8');
     expect(deleteFlyerResponse).toBeNull();
   });
-});
\ No newline at end of file
+});
+
+describe('Handle non-unique CategorySlugs across flyers in collection', () => {
```

Comment:
Keep the text inside of `describe` the same as you had before: `describe('getAllFlyerCategories tests', () => {`
What Archit meant was for the `test` part since each test case is testing for something different. At some point, we will need to refactor the entire testing suite and write test cases for resolvers.